src/parsers/fluent.js

```diff
@@ -18,30 +19,47 @@ export default class FluentParser {
     this.isValid = null;
   }
 
+  getLineAndColumnFromSpan(span) {
```

Comment:
This looks like a job for `lineOffset` and `columnOffset` exported by `fluent-syntax` :)

https://github.com/projectfluent/fluent.js/blob/fee9cbf347c4edecaaf60f757da6c515110f2356/fluent-syntax/src/index.js#L17-L36

Reply:
Ah, I didn't see those helpers yet. The code is quite similar, I'll use the upstream version then, thanks for the hint!